rpcs3/util/sysinfo.cpp

```diff
@@ -138,6 +138,33 @@ bool utils::has_fma4()
 	return g_value;
 }
 
+bool utils::has_erms()
+{
+	static const bool g_value = get_cpuid(0, 0)[0] >= 0x7 && (get_cpuid(7, 0)[1] & 0x200) == 0x200;
+	return g_value;
+}
+
+bool utils::has_fsrm()
+{
+	static const bool g_value = get_cpuid(0, 0)[0] >= 0x7 && (get_cpuid(7, 0)[3] & 0x10) == 0x10;
+	return g_value;
+}
+
+u32 utils::get_rep_movsb_threshold()
+{
+	static u32 g_value = 0xFFFFFFFF;
+	if (has_fsrm())
```

Comment:
Use a lambda otherwise there is no point in static initialization. Notice how others are const clarifing they are evaluated only once.